microsoftgraph/microsoft-graph-explorer-v4
# Feature: add locale to headers devxapi

## Overview

Adds 'Accept-Language' header and prepares the view for localized text from the server
Fixes #499 
Fixes #472 

### Demo
French:

![image](https://user-images.githubusercontent.com/58787602/81058456-24848980-8ed7-11ea-83d0-2dce4606adaa.png)


### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* Run the app
* Change url to preferred language like so `http://localhost:3000/fr-fr`
* Notice everything including samples and permissions change language
